contributing.md

```diff
@@ -1,117 +1,235 @@
-# Contributing to the Windows Community Toolkit
+# Contributing to the Windows Community Toolkit :sparkles::sparkles:
+
+Welcome to the Contributing page of the Windows Community Toolkit :tada:
+
+The entire Windows Community Toolkit team would like to say Congratulations for reaching this far and contributing to one of the most exciting and growing projects you have probably encountered. We are beyond excited for you to participate in the community. It is because of the valuable contributors like you that keep the Windows Community Toolkit thriving. Whether you fix any small bug or a major one, any contribution will go a long way to enhance the project. Our team will always be here if you may require any assistance regarding steps to submit the PR, review your work and any project related questions that you are unclear about.
+
+Once again Welcome aboard and have fun in this adventurous journey of Windows Community Toolkit :raised_hands:
+
+In the next few steps you will be able to learn how to contribute to this amazing community, but before you proceed any further it is good to understand what Widows Community Toolkit is all about. 
 
 The foundation of the **Windows Community Toolkit** is simplicity. 
 
-A developer should be able to quickly and easily learn to use the API. 
+A developer should be able to quickly and easily learn to use the API.
 
-Simplicity and a low barrier to entry are must-have features of every API. If you have any second thoughts about the complexity of a design, it is almost always much better to cut the feature from the current release and spend more time to get the design right for the next release. 
+Simplicity and a low barrier to entry are must-have features of every API. If you have any second thoughts about the complexity of the design, it is almost always much better to cut the feature from the current release and spend more time to get the design right for the next release.
 
 You can always add to an API, you cannot ever remove anything from one. If the design does not feel right, and you ship it anyway, you are likely to regret having done so.
 
 That's why many of the guidelines of this document are obvious and serve only one purpose: Simplicity.
 
- - [Questions](#question)
- - [Issues or Bugs](#issue)
- - [Submitting a pull request](#pr)
- - [Quality assurance for pull requests for XAML controls](#xaml)
- - [General rules](#rules)
- - [Naming conventions](#naming)
- - [Documentation](#documentation)
- - [Files and folders](#files)
+Here is how you can contribute to the Windows Community Toolkit. 
```

Comment:
We should probably start a new section of 'How to Contribute to the Community Toolkit' and list out the high-level things like reviewing docs, testing a PR, etc... and make some of the list below hidden behind sub-bullets or maybe we have a separate page all around actually contributing a feature to the Toolkit (as that's more involved)?

Reply:
WCT Wiki has solved this issue.